### DISCUSSION TITLE
Call command on binary_sensor

### DISCUSSION THREAD

**Discussion by thomasdg** [fuatakgun/eufy_security#1142] [Q&A]
Does anyone know how to call the command that is listed in the home assistant device Developer Tools > States list on a binary_sensor? I would love to be able to call the "open" command on my smartdrop. Maybe it needs some more development on the side of this HACS module, but if not: how to use it please? Thanks for any nudge in the right direction!

<img width="1341" alt="Screenshot 2024-05-04 at 14 09 13" src="https://github.com/fuatakgun/eufy_security/assets/5465002/a7f6c94f-f084-44a8-8e57-150776c1f2da">

**Comment by Clem-** [comment]
You may want to ask on this repo https://github.com/bropat/eufy-security-client that is what the integration is using underneath.
Last changelog says that it is compatible with `Smart Drop S300 (T8790)` but maybe the open action is not yet available.

**Comment by thomasdg** [comment]
His reaction in the past -when 3.0 came out- was that it is up to the integration in Home Assistant to get it connected. As the home assistant shows the "commands", I guess it does realise there is something available but commands are listed on the binary_sensor which is not supposed to have those :)
Have been looking in the code of this HACS module but I am not smart enough to get it called :)
